modules/genomic_browser/php/view_genomic_file.class.inc

```diff
@@ -64,7 +63,9 @@ class View_Genomic_File extends \NDB_Form
 
         if (!empty($this->candID) || !empty($this->genomic_file_id) ) {
             $this->_setFileData();
-            $this->tpl_data['has_permission'] = ($this->_hasAccess()) ? true : false;
+            $user = User::singleton();
+            $this->tpl_data['has_permission']
+                = ($this->_hasAccess($user)) ? true : false;
```

Comment:
It's not a big deal but... can this be changed to be

`$this->tpl_data['has_permission'] = $this->hasAccess($user);`

?

Reply:
👍 Good catch.. I was just basing it on what was there before without looking at it